front/app/modules/commercial/id_cow/messages.ts

```diff
@@ -49,4 +49,8 @@ export default defineMessages({
     id: 'app.modules.id_cow.cancel',
     defaultMessage: 'Cancel',
   },
+  helpImageAltText: {
+    id: 'app.modules.id_cow.helpImageAltText',
+    defaultMessage: 'Help',
```

Comment:
Can we try to make this a little more descriptive? 

"Image that shows where to find the ID serial number on an identity card."